Sources/mParticle-Appboy/MPKitAppboy.m

```diff
@@ -425,37 +426,105 @@ - (MPKitExecStatus *)routeCommerceEvent:(MPCommerceEvent *)commerceEvent {
         NSString *currency = commerceEvent.currency ? : @"USD";
         NSMutableDictionary *properties;
         
-        for (MPProduct *product in products) {
-            // Add relevant attributes from the commerce event
-            properties = [[NSMutableDictionary alloc] init];
-            if (baseProductAttributes.count > 0) {
-                [properties addEntriesFromDictionary:baseProductAttributes];
+        // Add relevant attributes from the commerce event
+        properties = [[NSMutableDictionary alloc] init];
+        if (baseProductAttributes.count > 0) {
+            [properties addEntriesFromDictionary:baseProductAttributes];
+        }
+        
+        if (_configuration[bundleProductsWithCommerceEvents] && [_configuration[bundleProductsWithCommerceEvents] boolValue]) {
```

Comment:
```suggestion
        if ([_configuration[bundleProductsWithCommerceEvents] boolValue]) {
```

This should be able to be simplified to only check the `boolValue` since if the configuration doesn't exist (aka is nil), then `boolValue` should be `NO`.